src/Sentry/Platforms/Native/SentryNativeBridge.cs

```diff
@@ -0,0 +1,276 @@
+using Sentry.Extensibility;
+
+namespace Sentry.Native;
+
+// https://github.com/getsentry/sentry-unity/blob/3eb6eca6ed270c5ec023bf75ee53c1ca00bb7c82/src/Sentry.Unity.Native/SentryNativeBridge.cs
+
+/// <summary>
+/// P/Invoke to `sentry-native` functions.
+/// </summary>
+/// <see href="https://github.com/getsentry/sentry-native"/>
+internal static class SentryNativeBridge
+{
+    public static bool Init(SentryOptions options)
+    {
+        _isLinux = RuntimeInformation.IsOSPlatform(OSPlatform.Linux);
+
+        var cOptions = sentry_options_new();
+
+        // Note: DSN is not null because options.IsValid() must have returned true for this to be called.
+        sentry_options_set_dsn(cOptions, options.Dsn!);
```

Comment:
That's no longer true, outside of Unity land.

Reply:
this point made me change my mind about keeping the unused native bindings code here. I've removed it so that it gets merged and reviewed properly  when actually moved here.